database/cassandra/cassandra.go

```diff
@@ -136,11 +138,19 @@ func (p *Cassandra) Run(migration io.Reader) error {
 	if err != nil {
 		return err
 	}
-	// run migration
+	// split multiple statements and run migration
 	query := string(migr[:])
-	if err := p.session.Query(query).Exec(); err != nil {
-		// TODO: cast to Cassandra error and get line number
-		return database.Error{OrigErr: err, Err: "migration failed", Query: migr}
+	matches := regexp.MustCompile(`(?m:;$)`).Split(query, -1)
```

Comment:
How about adding a few unit tests to make sure that this works as expected?

Reply:
```